contracts/injective-cosmwasm-mock/Cargo.toml

```diff
@@ -30,24 +30,22 @@ optimize = """docker run --rm -v "$(pwd)":/code \
 """
 
 [dependencies]
+cosmos-sdk-proto   = { version = "0.20.0", default-features = false }
+cosmwasm-schema    = { version = "1.5.0" }
 cosmwasm-std       = { version = "1.5.0", features = [ "abort", "cosmwasm_1_2", "cosmwasm_1_3", "cosmwasm_1_4", "iterator", "stargate" ] }
 cw-storage-plus    = { version = "1.2.0" }
 cw2                = { version = "0.16.0" }
 injective-cosmwasm = { path = "../../packages/injective-cosmwasm" }
 injective-math     = { path = "../../packages/injective-math" }
-injective-protobuf = { path = "../../packages/injective-protobuf"}
-schemars           = { version ="0.8.16"}
+injective-protobuf = { path = "../../packages/injective-protobuf" }
+injective-std      = { path = "../../packages/injective-std" }
+prost              = { version = "0.12.3" }
+protobuf           = { version = "3.3.0" }
+schemars           = { version = "0.8.16" }
 serde              = { version = "1.0.196", default-features = false, features = [ "derive" ] }
 thiserror          = { version = "1.0.56" }
-protobuf           = { version = "3.3.0" }
-prost              = { version = "0.11.9" }
-injective-std      = { version = "0.1.5" }
-cosmos-sdk-proto   = { version = "0.20.0", default-features = false }
-cosmwasm-schema    = { version = "1.5.0" }
-
-
 
 [dev-dependencies]
-injective-test-tube = { version = "1.1.7" }
-injective-std       = { version = "0.1.5" }
+injective-std       = { path = "../../packages/injective-std" }
+injective-test-tube = { path = "/Users/markuswaas/work/injective/test-tube/packages/injective-test-tube" }
```

Comment:
injective-test-tube = { path = "/Users/markuswaas/work/injective/test-tube/packages/injective-test-tube" } needs revert to injective-test-tube = { version = "1.1.7" }